src/test/scala/org/zalando/nakadi/client/KlientBuilderSpec.scala

```diff
@@ -8,21 +8,33 @@ import org.scalatest.{WordSpec, Matchers}
 class KlientBuilderSpec extends WordSpec with Matchers {
 
   "A Klient builder" must {
-    "must build a Klient instance, if everything is set properly" in {
+
+    // These things get tested by other things, anyways, so I'm not sure it's useful to test it (in development phase,
+    // it's probably been useful). AKa280116
+    //
+    "build a Klient instance, if everything is set up properly" in {
```

Comment:
The "must" need not be repeated in the individual test messages. 
Note (slightly unrelated): it is really important what test messages say, because they may be read by not only programmers (this is one of the aspects we're trying to improve in Helsinki - anyone should be able to understand, what was tested, what failed etc.)